apps/bfd-pipeline/bfd-pipeline-rif-to-rda/src/main/java/gov/cms/bfd/pipeline/bridge/CommandArguments.java

```diff
@@ -0,0 +1,264 @@
+package gov.cms.bfd.pipeline.bridge;
+
+import com.google.common.annotations.VisibleForTesting;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Deque;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+/** Helper class to parse and manage the command line arguments. */
+public class CommandArguments {
```

Comment:
isn't the team already using `org.apache.commons`, in which case would this provide the same functionality as this class? https://commons.apache.org/proper/commons-cli/

Reply:
I was thinking it could be made to utilize the `ConfigLoader` at some point so that the app could run via command line parameters or via yaml configurations, etc.  The current PR just doesn't include adding a common utils package, so we'd want to add that in a separate ticket.